### PR TITLE
Initial commit for Makefile

### PR DESCRIPTION
**Pull request by blckmn**

### CODE REVIEW

**Comment by DieHertz** [comment]
In my experience it's necessary to compile everything with maximum optimization level for ESC and debug it that way, otherwise ESC starts acting completely differently

**Comment by DieHertz** [comment]
The F0 chips I use have 64K btw :)

File: src/main/main.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with Betaflight-ESC.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+int main(void)
+{
+    return 0;
```
Comment (mikeller) [inline src/main/main.c]: Does this do Dshot1200? ;-P

File: make/local.mk [hunk 2]
```diff
@@ -0,0 +1 @@
+GCC_REQUIRED_VERSION := 6.3.1
```
Comment (mikeller) [inline make/local.mk]: I suspect this shouldn't be in here, but the file should be listed in `.gitignore`.

File: src/main/main.c (hunk 1 above)
Comment (DieHertz) [reply src/main/main.c]: Sure, it's implemented inside `stdint` :-P

**Comment by Linjieqiang** [comment]
Let's go and commit esc code.

**Comment by blckmn** [comment]
I'd say this is mergeable - and can be cleaned up as we go.

File: src/main/main.c (hunk 1 above)
Comment (blckmn) [reply src/main/main.c]: DSHOT2400! 👍

**Pull request merged**